process/datadog_checks/process/process.py

```diff
@@ -268,11 +269,16 @@ def get_process_state(self, name, pids, try_sudo):
             st['thr'].append(self.psutil_wrapper(p, 'num_threads', None, try_sudo))
 
             cpu_percent = self.psutil_wrapper(p, 'cpu_percent', None, try_sudo)
+            cpu_count = psutil.cpu_count()
             if not new_process:
                 # psutil returns `0.` for `cpu_percent` the first time it's sampled on a process,
                 # so save the value only on non-new processes
                 st['cpu'].append(cpu_percent)
-
+                if cpu_count > 0:
+                    st['cpu_norm'].append(cpu_percent/cpu_count)
+                else:
+                    self.log.debug('could not calculate the normalized \
+                                    cpu pct, cpu_count: %s' %(cpu_count))
```

Comment:
Let's move over to `str.format(...)`, and I think since the string is surrounded by parentheses, you can do something like:
```
self.log.debug("could not calculate the normalized "
               "cpu pct, cpu_count: {}".format(cpu_count))
```